modules/hosted-control-planes-troubleshooting.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-troubleshooting_{context}"]
+= Troubleshooting hosted control planes
+
+When you need to troubleshoot an issue with hosted control plane clusters, you can gather information by running the `hypershift dump cluster` command. The command generates output for the management cluster and the hosted cluster.
+
+The output for the management cluster contains the following content:
+
+* Cluster-scoped resources: These resources are node definitions of the management cluster.
+* The `hypershift-dump` compressed file: This file is useful if you need to share the content with other people.
+* Namespaced resources: These resources include all of the objects from the relevant namespaces, such as config maps, services, events, and logs.
+* Network logs: These logs include the OVN northbound and southbound databases and the status for each one.
+* HostedClusters: This level of output involves all of the resources inside of the hosted cluster.
+
+The output for the hosted cluster contains the following content:
+
+* Cluster-scoped resources: These resources include all of the cluster-wide objects, such as nodes and CRDs.
+* Namespaced resources: These resources include all of the objects from the relevant namespaces, such as config maps, services, events, and logs.
+
+Although the output does not contain any secret objects from the cluster, it can contain references to the names of secrets.
+
+.Prerequisites
+
+* You must have `cluster-admin` access to the management cluster.
+
+* You need the `name` value for the `HostedCluster` resource and the namespace where the CR is deployed.
+
+* You must have the `hcp` command line interface installed. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#hosted-install-cli[Installing the hosted control planes command line interface].
+
+* You must have the OpenShift CLI (`oc`) installed.
+
+* You must ensure that the `kubeconfig` file is loaded and is pointing to the management cluster.
+
+.Procedure
+
+* To gather output for troubleshooting, enter the following commands:
++
+[source,terminal]
+----
+CLUSTERNAME="samplecluster"
+CLUSTERNS="clusters"
+
+mkdir clusterDump-${CLUSTERNS}-${CLUSTERNAME}
+hypershift dump cluster \
```

Comment:
These commands are missing the "$" prompt. Also, there should only be a single command in each code block, so these should be broken up into multiple code blocks.

Same for the other code blocks below